kore/src/Kore/Log/WarnClaimRHSIsBottom.hs

```diff
@@ -0,0 +1,37 @@
+{- |
+Copyright   : (c) Runtime Verification, 2021
+License     : BSD-3-Clause
+-}
+module Kore.Log.WarnClaimRHSIsBottom (
+    WarnClaimRHSIsBottom (..),
+    warnClaimRHSIsBottom,
+) where
+
+import Kore.Attribute.SourceLocation
+import Kore.Rewrite.ClaimPattern
+import Log
+import Prelude.Kore
+import Pretty (
+    Pretty,
+ )
+import qualified Pretty
+
+newtype WarnClaimRHSIsBottom = WarnClaimRHSIsBottom {claim :: ClaimPattern}
+    deriving stock (Show)
+
+instance Pretty WarnClaimRHSIsBottom where
+    pretty (WarnClaimRHSIsBottom rule) =
+        Pretty.hsep
+            [ "The right-hand side of the claim is bottom:"
+            , Pretty.pretty (from rule :: SourceLocation)
+            ]
+
+instance Entry WarnClaimRHSIsBottom where
+    entrySeverity _ = Warning
+    helpDoc _ = "warn when the right-hand side of a claim is bottom"
```

Comment:
I think we should be giving `oneLineDoc` implementations to all new log entries. That feature probably post-dates the entry that you used as a template for this one. The `oneLineDoc` should just be the prettified source location with no other text. See for example `DebugBeginClaim`.

Reply:
I added the function, but I don't know how to test it. I thought I had to pass `--log-format oneline` to `kprove`, but `kprove` doesn't seem to forward it to `kore-exec`.

Reply:
If you run `kprove` with the following option `--haskell-backend-command "/path/to/kore-exec --log-format online"` it should work.

Reply:
Thanks, this worked! Would it be a good idea to add implementations of `oneLineDoc` for all existing entries in this PR, or should I open a fresh issue about that?

Reply:
I think it's better if we do that separately, so please open an issue. Thanks!